assets/behaviors/creatures/combinedCritter.behavior

```diff
@@ -0,0 +1,107 @@
+{
+selector: [
+  {
+    guard: {
+
+      // Territory would to be implemented as Component
+      // Implementation of territorial Critter
+      componentPresent: "Behaviours:Territory",
+      values: ["F Territory != null"],
```

Comment:
This would try to make a lookup of a _field_ (indicated by `F`) named `null`.

However, I believe you want to check against the _value_ `null`, thus using `V` would be correct.

```suggestion
      values: ["V Territory != null"],
```

Alternatively, you can use the `exists` operator (basically `... != null`) without an explicit value to check against:

```suggestion
      values: ["N Territory exists"],
```

Reply:
With the comment above about the component, this should most likely be 
```suggestion
      values: ["F location exists"],
```